src/common/options.cc

```diff
@@ -5521,6 +5521,13 @@ std::vector<Option> get_rgw_options() {
     .set_default(0)
     .set_description("Delay after processing of bucket listing chunks (i.e., per 1000 entries) in milliseconds"),
 
+    Option("rgw_lc_max_worker", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(3)
```

Comment:
I don't think that 3 is a good number because we have 32 shards, maybe 2 or 4?

Reply:
why, exactly?

Reply:
Because we will want theoretically spread equally the lockings across rgws (32/3 not integer). one rgw will be with less lockings

Reply:
seems we shouldn't make too much of the 32 value, it could be any number